app/jobs/create_member_email_batch_job.rb

```diff
@@ -0,0 +1,24 @@
+class CreateMemberEmailBatchJob < Struct.new(:sender_id, :community_id, :subject, :content, :locale)
+
+  include DelayedAirbrakeNotification
+
+  # This before hook should be included in all Jobs to make sure that the service_name is
+  # correct as it's stored in the thread and the same thread handles many different communities
+  # if the job doesn't have host parameter, should call the method with nil, to set the default service_name
+  def before(job)
+    # Set the correct service name to thread for I18n to pick it
+    ApplicationHelper.store_community_service_name_to_thread_from_community_id(community_id)
+  end
+
+  def perform
+    sender = Person.find(sender_id)
+    current_community = Community.find(community_id)
+
+    Delayed::Job.transaction do
+      current_community.members.each do |recipient|
+        Delayed::Job.enqueue(CommunityMemberEmailSentJob.new(sender.id, recipient.id, community_id, subject, content, locale))
+      end
+    end
```

Comment:
I'm quite confused about this.
transaction creates only one connection to the db, but inside we need to "actions": insert row and enqueue